Fitness/Controllers/OnboardingGymsViewController.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  OnboardingGymsViewController.swift
+//  Fitness
+//
+//  Created by Phillip OReggio on 3/5/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    // MARK: - INITIALIZATION
+    var titleLabel: UILabel!
+    var nextButton: UIButton!
+    var nextButtonArrow: UIImageView!
+    var backButton: UIButton!
+    var backButtonArrow: UIImageView!
+    
+    // TableView
+    var gymsTableView: UITableView!
+    let reuseIdentifier = "gymsTableView"
```

Comment:
Maybe add this as a static field of the cell to which it refers (it's done this way with the other reusable cells/headers we have)? This way having several cell types is simpler and it is more clear which cell type the reuse identifier refers to

Reply:
Ok, will change!